ui/src/containers/Header/__snapshots__/Header.test.tsx.snap

```diff
@@ -19,43 +19,82 @@ exports[`Header should render the header component and find Search component 1`]
+                                </Icon>
+                              </div>
+                            </GridItem>
+                          </div>
+                        </Grid>
                       </div>
                     </GridItem>
-                    <GridItem span={1} onClick={[Function: onClick]} className=\\"header-search-hint\\">
-                      <div className=\\"pf-l-grid__item pf-m-1-col header-search-hint\\" onClick={[Function: onClick]}>
-                        <Icon id=\\"help\\" size=\\"sm\\" label=\\"search-tips\\">
-                          <HelpIcon size=\\"sm\\" label=\\"search-tips\\" color=\\"currentColor\\" noVerticalAlign={false}>
-                            <svg style={{...}} fill=\\"currentColor\\" height=\\"1em\\" width=\\"1em\\" viewBox=\\"0 64 1024 1024\\" aria-labelledby={{...}} aria-hidden={true} role=\\"img\\" label=\\"search-tips\\">
-                              <path d=\\"M512.059-73.143c-282.338 0-512.059 229.673-512.059 512.025 0 282.235 229.721 511.975 512.059 511.975 282.281 0 511.941-229.735 511.941-511.975 0.005-282.352-229.659-512.025-511.941-512.025zM512.059 826.523c-213.826 0-387.728-173.856-387.728-387.643 0-213.89 173.904-387.694 387.728-387.694 213.717 0 387.671 173.803 387.671 387.694 0.005 213.785-173.957 387.643-387.671 387.643zM585.143 164.571v109.714c0 4.951-1.808 9.237-5.429 12.857s-7.906 5.429-12.857 5.429h-109.714c-4.953 0-9.239-1.808-12.857-5.429s-5.429-7.906-5.429-12.857v-109.714c0-4.951 1.81-9.237 5.429-12.857s7.904-5.429 12.857-5.429h109.714c4.951 0 9.237 1.808 12.857 5.429s5.429 7.906 5.429 12.857zM521.616 365.714c118.343 0 214.286 88.965 214.286 187.429s-95.943 178.286-214.286 178.286c-173.045 0-208.091-93.714-213.963-171.113 0-7.936 6.729-9.838 15.872-9.838s108.073 0 113.143 0c6.096 0 14.475 0.633 17.554 10.571 0 48.857 135.968 54.953 135.968-7.904 0-31.506-29.717-63.817-68.571-66.194s-82.286-7.607-82.286-54.199c0-13.022 0-25.673 0-44.693 0-19.015 9.717-22.343 27.431-22.343s54.853-0.002 54.853-0.002z\\" transform=\\"rotate(180 0 512) scale(-1 1)\\" />
-                            </svg>
-                          </HelpIcon>
-                        </Icon>
+                    <GridItem span={2} onClick={[Function: onClick]} className=\\"header-add-resource\\">
+                      <div className=\\"pf-l-grid__item pf-m-2-col header-add-resource\\" onClick={[Function: onClick]}>
+                        <Grid>
+                          <div className=\\"pf-l-grid\\">
+                            <GridItem offset={2} className=\\"hub-header-contribute-margin\\">
+                              <div className=\\"pf-l-grid__item pf-m-offset-2-col hub-header-contribute-margin\\">
+                                <Text component=\\"h3\\">
+                                  <h3 data-pf-content={true} className=\\"\\">
+                                    <span className=\\"hub-header-contribute\\">
+                                      Contribute
```

Comment:
This can create confusion like is it about contributing to Hub code or contributing a resource

I think it can be link `Add a resource` or other suggestions are welcome

Reply:
> This can create confusion like is it about contributing to Hub code or contributing a resource
> 
> I think it can be like `Add a resource` or other suggestions are welcome

cc: @sm43 @PuneetPunamiya @vinamra28

Reply:
maybe we can have a dropdown for now when we click on contribute and then we can add `Submit a Task` as one item and when we have pipelines support then we can have `Submit a Pipeline` as well since submission guidelines would be different for both. WDYT?

Reply:
Nope I think `Add a Resource` should be enough because be it a pipeline or a task or any other resource ultimately catalog will be the only source of truth for hub,  where in TEP and hub docs they get a complete reference as well for submitting a resource to see it on Hub

Reply:
I think there would be same instructions for adding tasks/pipelines to Hub and keeping `Add a Resource` sounds good , can we have an alternative short name e.g. `Share` or any better instead of long words(Add a Resource) because it would take a bit more space in header  , WDYT?

Reply:
Being very naive over here... will not `Add a Resource` look big in the Header?